.eslintrc.json

```diff
@@ -8,7 +8,8 @@
       2,
       {
         "typeSeparator": "dot",
-        "nameStyle": "dash"
+        "nameStyle": "dash",
+        "ignoreTypeSuffix" : true
```

Comment:
@jtormey @plondon @prwelber  I added `ignoreTypeSuffix` so that a class name can be `BlahController` or `BlahService`, but I suggest we only do that for controllers.